src/main/java/teammates/storage/entity/StudentProfile.java

```diff
@@ -18,6 +19,11 @@
 @PersistenceCapable
 public class StudentProfile {
 
+    // CHECKSTYLE.OFF:JavadocVariable self-documenting
+    @NotPersistent
+    public static final String PRIMARY_KEY_NAME = Entity.getFieldWithPrimaryKeyAnnotation(StudentProfile.class);
```

Comment:
It is indeed neater; however, would it have performance costs? The app should not be made to run the reflection code every time the primary key name is required, especially since we're using it to implement a method that's meant to be as efficient as possible.